src/content/docs/change-tracking/change-tracking-introduction.mdx

```diff
@@ -20,13 +20,30 @@ Change tracking allows you to capture deployments and other changes on any part
 
 Our change analysis UIs provide details and insights about each of the changes you track, such as faceted errors, log attribute trends, related issues and anomalies, and calculated impacts on key signals. With these insights, you'll be able to understand the impact of changes on system performance and quality. Change tracking metadata, such as commit SHAs, changelogs, and deep links, will help you resolve issues more quickly.
 
-To get acquainted with various aspects of change tracking, watch the video below or see our [feature highlights](#feature-highlights). If you're ready to capture and analyze changes across your stack, skip to the steps about how to [start tracking changes](#start-tracking).
+To get acquainted with various aspects of change tracking, watch the video below or see our [feature highlights](#feature-highlights). If you're ready to capture and analyze changes across your stack, go right to [Start tracking changes](#start-tracking).
 
 <Video id="ikRUKJf7hyU" type="youtube" />
 <figcaption>
   Here's a six-minute video explaining change tracking.
 </figcaption>
 
+## Start tracking your changes [#start-tracking]
+
+To begin capturing and analyzing changes, make sure you have set up a free New Relic account.
+<InlineSignup/>
+
+Once you have an account, choose one of these options to get started.
+
+* [NerdGraph (GraphQL) API](/docs/change-tracking/change-tracking-graphql): You can use GraphQL mutations to designate what changes to track and then run queries to list details about what you're tracking.
```

Comment:
Do we need to link to a resource that explains "GraphQL mutations"? I have no clue what this concept means.